packages/ketcher-react/src/script/ui/dialog/toolbox/sgroup.module.less

```diff
@@ -14,48 +14,73 @@
  * limitations under the License.
  ***************************************************************************/
 @import "src/style/variables";
-
+@import "src/style/mixins";
+// stylelint-disable selector-pseudo-class-no-unknown
+// stylelint-disable no-descending-specificity
 .sgroup {
   width: @width-small;
+  border-radius: 6px;
+  background-color: @color-background-primary;
+
+  header {
+    border-bottom: 1px solid @color-grey-3;
+    margin: 0;
+    padding: 0 12px;
+    height: 36px;
+    font-weight: 500;
+  }
+
+  & > div[class^='Dialog-module_dialog_body'], footer {
```

Comment:
Maybe, dialog body styles can be redefined in Dialog module styles (if are same in all dialogs);
it's safer to not use 'class^='Dialog-module_dialog_body'', 'div' selector would be enough here

Reply:
removed this style